bedrock/firefox/templates/firefox/testflight.html

```diff
@@ -1,131 +1,138 @@
+<main role="main">
+  {% call hero(
+  title=_('Help shape the future <br>of Firefox for iOS'),
+  desc=_('Sign up to test pre-release versions to make Firefox even better.'),
+  class='mzp-t-firefox mzp-t-dark mzp-t-product-firefox',
+  include_cta=False,
+  heading_level=1) %}
+  {% endcall %}
+
+  <aside class="mzp-l-content">
+    <div class="mzp-c-newsletter">
+      <div class="mzp-c-newsletter-image">
+        {{ high_res_img('img/home/2018/newsletter-graphic.png', {'alt': ''}) }}
+      </div>
+
+      <form id="newsletter-form" class="mzp-c-newsletter-form" action="{{ url('newsletter.subscribe') }}" method="post"
+        data-spinner-color="#fff">
+
+        {{ newsletter_form.newsletters|safe }}
+        {# test flight program is only available in english #}
+        <input type="hidden" name="lang" value="en">
+        <input type="hidden" name="source_url" value="{{ request.build_absolute_uri() }}">
+
+        <fieldset class="mzp-c-newsletter-content">
+          <div class="mzp-c-form-errors" id="newsletter-errors"></div>
+
+          <p>
             <label for="id_email">{{ _('Email') }}</label>
             {{ newsletter_form.email|safe }}
-          </div>
-          <button class="button button-dark" id="footer_email_submit" type="submit">Submit</button>
-        </div>{#-- /#email-submit-wrapper --#}
-
-        <div id="terms-format-wrapper">
-          <div id="format-wrapper">
-            {{ newsletter_form.fmt|safe }}
-          </div>
-
-          <label class="terms-check-label" for="id_terms">
-            <input id="id_terms" name="terms" required="required" type="checkbox">
-            <span class="title">
-            {% trans url='#terms-conditions' %}
-              I have read and agree to these
-              <a href="{{ url }}">Terms and Conditions</a>
-            {% endtrans %}
-            </span>
-          </label>
-        </div>{#-- /#terms-format-wrapper --#}
-
-        {{ newsletter_form.privacy|safe }}
-      </div>{#-- /#form-fields --#}
+          </p>
 
-      <div id="newsletter-spinner" class="hidden"></div>
-    </form>
+          <fieldset class="mzp-u-inline">
+            <legend>Format</legend>
+            <p>{{ newsletter_form.fmt|safe }}</p>
+          </fieldset>
 
-    <div id="newsletter-form-thankyou" class="thank newsletter-form newsletter-form-extra">
-      <h2>{{ _('Congrats!') }}</h2>
-
-      <h3>{{ _('You have been added to the Firefox for iOS testing program.') }}</h3>
-
-      <p>
-      {% trans %}
-        When test spots are available, you will receive an invitation from
-        TestFlight, Apple's App Store program that manages pre-release
-        software, to participate in the program.
-      {% endtrans %}
-
-        <strong>{{ _('Please accept it.') }}</strong>
-      </p>
-
-      <p>{{ _('Be aware that this email may be in your spam folder.') }}</p>
+          <p>
+            <label for="privacy" class="mzp-u-inline">
```

Comment:
The `for` attribute should map to the id of the input here?
```suggestion
            <label for="id_terms" class="mzp-u-inline">
```